### PR TITLE
Create Q20_print_diamond_pattern.cpp

### PR DESCRIPTION
**Pull request by vaibhav19mark1**
This program displays the pattern in diamond shape according to the value entered.

### CODE REVIEW

**Review by drdeepakuniyal — CHANGES_REQUESTED** [review]
Please check your file name and also leave some comments for users - like, where your program will work and situations in which it might fail.

**Comment by vaibhav19mark1** [comment]
> Please check your file name and also leave some comments for users - like, where your program will work and situations in which it might fail.

Sir it will not fail whether the value of n is odd or even

**Comment by drdeepakuniyal** [comment]
please change your file names as per instructions and leaving comments means inside your file, not on GitHub.

File: Q20_pattern_diamond.cpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#include<iostream>
+using namespace std;
+int main(){
+    int n;
+    cout<<"enter number of rows: ";
+    cin>>n;
```
Comment (vaibhav19mark1) [inline Q20_pattern_diamond.cpp]: Enter the number of rows for printing the diamond

**Review by drdeepakuniyal — APPROVED** [review]
Good Job!

**Pull request merged**